batchupload/make_info.py

```diff
@@ -69,7 +69,7 @@ def load_data(self, in_file):
         pass
 
     @abstractmethod
-    def load_mappings(self, update=True):
+    def load_mappings(self, update):
```

Comment:
Is there  a reason why we're not forcing an update anymore? I'm thinking maybe it should be set to False by default, in case we ever want to avoid overwriting existing mappings